src/PublicApiGenerator/MethodNameBuilder.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.CodeDom;
+using System.Linq;
+using Mono.Cecil;
+using static System.String;
+
+namespace PublicApiGenerator
+{
+    public static class MethodNameBuilder
+    {
+        public static string AugmentMethodNameWithMethodModifierMarkerTemplate(MethodDefinition methodDefinition,
+            MemberAttributes attributes)
+        {
+            var name = CSharpOperatorKeyword.Get(methodDefinition.Name);
+
+            if (methodDefinition.DeclaringType.IsInterface)
+            {
+                return name;
+            }
```

Comment:
ℹ This is going to have to change in the future to support the many modifiers now allowed on interface members in C#.
https://docs.microsoft.com/en-us/dotnet/csharp/whats-new/csharp-8#default-interface-methods

Reply:
Don't get me started... 😂
Yeah but we can focus on that in another major release